asyncpg/connection.py

```diff
@@ -1215,6 +1204,13 @@ def _drop_global_statement_cache(self):
         with self._stmt_exclusive_section:
             return await self._do_execute(query, executor, timeout)
```

Comment:
Maybe we should refactor `def _execute` to:

```python
async def _execute(self, query, args, limit, timeout, return_status=False):
    with self._stmt_exclusive_section:
        return await self.__execute(self, query, args, limit, timeout, return_status)
```

?

Reply:
Sure. Makes sense.